CAD/monomakh-sapr.ksy

```diff
@@ -0,0 +1,34 @@
+meta:
+  id: chg
+  endian: le
+  file-extension: chg
+  application:  MONOMAKH-SAPR
+  title: CHG pakege files
```

Comment:
Given that you've already included "file-extension" and "application", I believe it sufficiently identifies the file. If you really want to add "title", then probably it should be something like "CHG package file".